diff --git a/include/mzpeak/util/enumerable_proxy.h b/include/mzpeak/util/enumerable_proxy.h
index 6fe6766..f256297 100644
--- a/include/mzpeak/util/enumerable_proxy.h
+++ b/include/mzpeak/util/enumerable_proxy.h
@@ -55,9 +55,9 @@ public:
 
     // Copy, move, and assignment constructors.
     Iterator(const Iterator&) = default;
-    Iterator(const Iterator&&) = default;
+    Iterator(Iterator&&) = default;
     Iterator& operator=(const Iterator&) = default;
-    Iterator& operator=(const Iterator&&) = default;
+    Iterator& operator=(Iterator&&) = default;
 
     /// Prefix increment.
     Iterator& operator++()
